andela/ah-orcas-frontend
# #162439002 beautify navbar and footer

#### What does this PR do?
```It makes the navbar and footer look better```
#### Description of Task to be completed?
```This PR improves the look, color and feel of the navbar and the footer ```
#### How should this be manually tested?
- Clone this repository 
```git clone https://github.com/andela/ah-orcas-frontend.git```
- Change the working directory to the project folder
``` cd ah-orcas-frontend```
- Checkout the setup redux branch
``` git checkout ft-navbar-footer-beautify ```
- Install the required dependencies:
```npm install```
- Run the local development server:
```npm start```
- You should be able to view the ah-orcas-frontend-application on your browser

#### What are the relevant pivotal tracker stories?
[#162439002](https://www.pivotaltracker.com/story/show/162439002)
#### Screenshots (if appropriate)
![screenshot 2018-12-06 at 06 59 12](https://user-images.githubusercontent.com/8406752/49560707-90e70c00-f924-11e8-941b-0f6e4fa8da66.png)

#### Checklist:
